### DISCUSSION TITLE
kubernetes yaml standards

### DISCUSSION THREAD

**Discussion by Staggerlee011** [BretFisher/ama#170] [Q&A]
do you have any naming standards that you follow to manage your yaml files.  ie services are always <app>-service, cant see any blog posts about it but does look like as your cluster grows it could get very confusing without. 

do you have any label standards as well :)? tier, app, version seem to be common but have you found others that you like to include?

**Comment by BretFisher** [comment]
I tend to find every team has its own existing standards before they implemented Docker/Kubernetes that we just add as labels or annotations.

More standards info:

Container labels: https://github.com/opencontainers/image-spec/blob/master/annotations.md

Kubernetes labels, annotations, and taints:
https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#motivation
https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/
